public/video-ui/src/components/Workflow/Workflow.js

```diff
@@ -46,13 +46,18 @@ class Workflow extends React.Component {
         video={video}
         workflowSections={this.props.workflow.sections || []}
         workflowStatuses={this.props.workflow.statuses || []}
-        workflowPriorities={this.props.workflow.priorities}
+        workflowPriorities={
+          this.props.workflow.priorities.map(({ name, value }) => {
+            return { id: value.toString(), title: name };
+          }) || []
+        }
```

Comment:
Additionally I moved this transformation code to here and out of `WorkflowForm` so that it's all happening at the same level

Reply:
👍